.github/workflows/docs_build_and_deploy.yml

```diff
@@ -12,23 +12,35 @@ on:
     tags:
       - '*'
   pull_request:
+  merge_group:
   workflow_dispatch:
 
 jobs:
+  linting:
+    # scheduled workflows should not run on forks
+    if: (${{ github.event_name == 'schedule' }} && ${{ github.repository_owner == 'neuroinformatics-unit' }} && ${{ github.ref == 'refs/heads/main' }}) || (${{ github.event_name != 'schedule' }})
```

Comment:
Cool! I did something [similar](https://github.com/neuroinformatics-unit/datashuttle/blob/main/.github/workflows/code_test_and_deploy.yml) recently but didn't realise it would run on forks, I need to update the datashuttle one! I think according to [this](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule) schedule jobs will run only on `main` , so you might be able to drop the last conditional (`${{ github.ref == 'refs/heads/main' }}`).  I saw [here](https://dev.to/hugovk/til-how-to-disable-cron-for-github-forks-2d0l) they use the conditional `${{ github.repository_owner == 'neuroinformatics-unit' || github.event_name != 'schedule' }}`, I think this is equivalent to the above but can loose one condition (`| (${{ github.event_name != 'schedule' }})`.

BTW at present this CI does not have a cron job schedule and so IIUC this conditional will never be used.